DEVELOPMENT.md

```diff
@@ -50,7 +50,7 @@ The script will output the location of the kubeconfig file that can be used to i
 ```
 
 ---
-**_NOTE:_**  podman is used per default as container engine if available on the machine. If both podman and docker are available it is possible to force the use of docker by setting an environment variable `CONTAINER_ENGINE=docker`.
+**_NOTE:_**  The podman is used per default as container engine if available on the machine. If both podman and docker are available it is possible to force the use of docker by setting an environment variable `CONTAINER_ENGINE=docker`.
```

Comment:
I don't think that the podman binary needs to be prefixed by `The`. I would revert that specific change. As a matter of fact, it is a repeat of the note line 29, so if possible I would reference that instead of duplicating the information.

Reply:
+1 on not having "The". I am not too concerned about the duplication. It is sometimes better not to break the flow.

Reply:
Ok got it